sdk/core/Azure.Core.TestFramework/src/TestRecording.cs

```diff
@@ -98,10 +98,15 @@ public TestRandom Random
                     switch (Mode)
                     {
                         case RecordedTestMode.Live:
+                            #if NET6_0_OR_GREATER
```

Comment:
Is this fork required or are you just trying to leverage some new .NET 6 code? Is it worth the fork?

Reply:
Some APIs are obsolete in .net 6, most forks that you see are to avoid build warnings.

Reply:
Nit: any reason not to align these with the start of the line like all other files?